src/main/java/org/cactoos/scalar/RetryScalar.java

```diff
@@ -31,15 +31,17 @@
  * Func that will try a few times before throwing an exception.
  *
  * <pre>
- * new RetryScalar<>(
- *     () -> {
- *         if (new SecureRandom().nextDouble() > 0.3d) {
- *         throw new IllegalArgumentException("May happen");
- *       }
- *       return 0;
- *     },
- *     5
- * ).value() // will try to run 5 times before throwing an exception
+ * {@code
+ *   new RetryScalar<>(
+ *       () -> {
+ *           if (new SecureRandom().nextDouble() > 0.3d) {
+ *           throw new IllegalArgumentException("May happen");
+ *         }
+ *         return 0;
+ *       },
+ *       5
+ *   ).value() // will try to run 5 times before throwing an exception
+ * }
```

Comment:
@Isammoc Please, correct the indentation of the comment above (`throw` and its outlying curly braces).

Reply:
@paulodamaso Done